client/components/gsuite/gsuite-new-user-list/index.tsx

```diff
@@ -36,7 +36,7 @@ const GSuiteNewUserList: FunctionComponent< GSuiteNewUserListProps > = ( {
 	showAddAnotherMailboxButton = true,
 	users = [],
 	validatedMailboxUuids = [],
-} ) => {
+}: GSuiteNewUserListProps ) => {
```

Comment:
I believe we need to specify a return type, which I think can be kept generic as `ReactElement`.
```suggestion
}: GSuiteNewUserListProps ): ReactElement => {
```